### ISSUE TITLE
question: disable the unicode symbol while displaying values

### ISSUE THREAD

**Issue by insomnimus** [elves/elvish#1378]
I'm using elvish as a scripting language on windows (powershell messes my pipelines and isn't comfortable to use as a scripting language)

I'm also a screen reader user (visually impaired).

So, i was wondering if it's possible to omit the `▶` symbol while displaying values, this doesn't work well with my screen reader software.

```èlvish
use str

str:join ':' [a b c d]
```

Displays:

```òutput
▶ a:b:c:d
```

I know it's just one symbol, however the screen reader reads it as "play button", which is not very ergonomic.

I also know i can `echo` it instead, but again not ergonomic.

So, is there a flag i can set to disable this behaviour?

__Edit: Fix markdown syntax.__

**Comment by krader1961** [comment]
See https://elv.sh/ref/builtin.html#value-out-indicator

```
> put yes
▶ yes
> set value-out-indicator = ''
> put no
no
> set value-out-indicator = 'value out is: '
> put value
value out is: value
```

**Comment by insomnimus** [comment]
Thank you, this is what i wanted!
